okapi-core/src/main/raml/TenantDescriptor.json

```diff
@@ -18,5 +18,6 @@
       "description": "Tenant description",
       "type": "string"
     }
-  }
+  },
+  "required" : [ "id" ]
```

Comment:
The "desciption" of "id" should mention "^[a-z][a-z0-9]{0,30}$" and that it cannot be `pg`